pkg/management/component_test.go

```diff
@@ -515,6 +523,14 @@ func TestCreateUser(t *testing.T) {
 
 		var onboardingStatus = "user-created-by-api"
 
+		var ctx = context.WithValue(context.Background(), cs.CtContextAccessToken, accessToken)
+		ctx = context.WithValue(ctx, cs.CtContextRealm, realmName)
```

Comment:
I think with your changes after the review, you do not need to set the context anymore